### PR TITLE
Fixed Malfunctioning of Asset Creation Page and revamped it's UI

### LINKED ISSUE #2841: Asset Creation Form Malfunction
- [ ] when we are creating an asset, the asset type is by default showing as internal, and when we submit the form without clicking anywhere in the asset type, this error occurs.
- [ ] when we click on top of the "Internal", no drop list will be shown, to see the drop list, we need to go to the arrow itself to see it.

### LINKED ISSUE #2842: Revamp Asset Creation Form
we are having a lot of blank white spaces in the asset creation form. revamp the UI to a better look

### PR DESCRIPTION
**Pull request by patelaryan7751**
Fixes #2841 
Fixes #2842 

# Bugs Fixed

- [x] Now the asset type is by default shown as "select" you need to choose the asset type then you can proceed
##
https://user-images.githubusercontent.com/59426397/175628907-ea007ca1-b58f-4e8b-8d05-63c512f02b0b.mp4
##

- [x] Now when you click on the button the drop list gets generated 
##
https://user-images.githubusercontent.com/59426397/175635201-5c26ae69-0180-45a3-bc15-c312278ab22e.mp4

## Revamped the asset creation form UI
image

### CODE REVIEW

**Review by cp-Coder — APPROVED** [review]
LGTM

**Review by cp-Coder — APPROVED** [review]

**Comment by nihal467** [comment]
@patelaryan7751 



- [ ] replace the location place with this three-button
- [ ] Change the is working heading to Working Status and create a droplist similar to assets where it asks working and not working
- [ ] make it fittable in a laptop screen and avoid scrolling on the page

**Comment by nihal467** [comment]
@patelaryan7751 what is the status

**Comment by patelaryan7751** [comment]
@nihal467 i am working on it my laptop had crashed

**Comment by patelaryan7751** [comment]
Hey, @nihal467 I have done the changes

asset manage

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/AssetCreate.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ const AssetCreate = (props: AssetProps) => {
   const [asset_class, setAssetClass] = useState<AssetClass>();
   const [not_working_reason, setNotWorkingReason] = useState("");
   const [description, setDescription] = useState("");
-  const [is_working, setIsWorking] = useState("0");
+  const [is_working, setIsWorking] = useState<string | undefined>("");
```
Comment (khavinshankar) [inline src/Components/Facility/AssetCreate.tsx]:
```suggestion
  const [is_working, setIsWorking] = useState<string | undefined>(undefined);
```

Initially if this not set to `undefined` the error won't be triggered when we submit the form without modifying is_working

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**